app/index.php

```diff
@@ -1,9 +1,30 @@
 <?php
-session_start();
 require "../vendor/autoload.php";
+session_start();
 $conn = new \App\DbConnector();
 $store = new \App\Store($conn->getDb());
-$books = $store->getAllBooks();
+
+if (isset($_GET['search'])) {
+    $searchTerm = $_GET["search"];
+    $searchGetTerm = 'search=' . $_GET['search'] . "&";
+    $searchTitle = new \App\SearchTitle($conn->getDb());
+    $books = $searchTitle->performSearch($searchTerm);
+    $sortBooks = new \App\SortBooks($books);
+} else {
+    $searchTerm = "";
+    $searchGetTerm = "";
+    $books = $store->getAllBooks();
+    $filteredBooksPrices = $store->getAllBookPrices();
+    $sortBooks = new \App\SortBooks($filteredBooksPrices);
+}
+
+if ((!empty($_GET) && isset($_GET['min'])) && $_GET['min'] && $_GET['max']) {
+    $books = $store->getBooksWithinRange($_GET['min'], $_GET['max'], $searchTerm);
```

Comment:
no validation on the min and max being numbers, if you change the url to a string, you get a big error!